schemas/io.snowplow.ecomm/add_payment_method/jsonschema/1-0-0

```diff
@@ -0,0 +1,28 @@
+{
+    "description": "Event occurs when a payment method is added during checkout.",
+    "properties": {
+        "payment_id": {
```

Comment:
Here I would be tempted to change this to `payment_type_id` . Since `payment` might be confused with the whole process of paying so someone might wrongly add something like the transaction Id or checkout Id.

Otherwise we can also add a description here and explain that if an 'id' is not available, they can use a friendly name for the payment method.

Reply:
I'll add an description to state that the ID here is for the payment entity that is attached to the same event. Since this isn't a payment type per se we should leave this field's naming with the convention we have everywhere else, to denote that ID points to the entities (the entities' name will form the field ID's name).